.eslintrc.js

```diff
@@ -88,7 +99,15 @@ module.exports = {
     'security/detect-object-injection': 'off',
     'security/detect-non-literal-fs-filename': 'off',
     'security/detect-non-literal-require': 'off',
-    'security/detect-unsafe-regex': 'off' // TODO: Add `eslint-plugin-regexp` and fix these issues.
+    'security/detect-unsafe-regex': 'off', // TODO: Add `eslint-plugin-regexp` and fix these issues.
+
+    // TODO: Discuss whether to enable these rules.
+    'unicorn/no-null': 'off',
+    'unicorn/prevent-abbreviations': 'off',
```

Comment:
I'd say it's fine to use `null`, especially explicitly :man_shrugging: 

I also don't feel a need to remove the abbreviations right now :man_shrugging: